as-controller-board-status/res/configuration.yaml

```
@@ -0,0 +1,34 @@
+# Copyright © 2023 Intel Corporation. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+
+Writable:
+  LogLevel: INFO
+
+Service:
+  Host: localhost
+  Port: 48094
+  StartupMsg: This EdgeX application service reads events from a controller board device service and propagates the information from the controller board to other services and API endpoints
+
```

Add Notification client which is used in the code.
```suggestion
Clients:
  support-notifications:
    Protocol = "http"
    Host = "localhost"
    Port = 59860
```

---

notification client is not in the common config? ok i'll add it

---

Nope, just core-metadata

---

thanks, added